### PR TITLE
fix(preview): Annotations header misalignment in IE11

### PR DESCRIPTION
**Pull request by ConradJChan**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @ConradJChan has signed the CLA. Thanks for the pull request!

File: src/elements/content-preview/preview-header/PreviewHeader.scss [hunk 1]
```diff
@@ -45,6 +45,7 @@
     .ba-point-post-cancel-container,
     .ba-draw-post-cancel-container {
         position: absolute;
+        top: 6px;
```
Comment (mxiao6) [inline src/elements/content-preview/preview-header/PreviewHeader.scss]: Will this affect the other browsers? Why the misalignment only happened in IE11, not the other browsers?

**Comment by ConradJChan** [comment]
@jstoffan - yeah, it looks good in the other browsers

File: src/elements/content-preview/preview-header/PreviewHeader.scss (hunk 1 above)
Comment (ConradJChan) [reply src/elements/content-preview/preview-header/PreviewHeader.scss]: Not sure, seems like the other browsers just handle it more gracefully

**Pull request merged**